ZupIT/beagle-web-react
# Allow text component to render not only strings

**- What I did**
As context will be able to change it's type on the web core project, I identified that the text component was not able to render the object values, crashing the application, so I changed the render method to be able to parse the data to a readable string

**- How I did it**
When the text attribute is changed, runs a validation of data type, then try to parse it as a string, rendering the text of the component as a readable string.  

**- How to verify it**
You can start a context with a string then set the context with a `path` and see the value change, you can use the following json to test:

https://run.mocky.io/v3/3797bd95-5837-4225-8b7a-edefbbed0f5f
